docs/adr/0002-tarball-hosting.md

```diff
@@ -0,0 +1,84 @@
+# 1) Use Github for hosting release tarballs {#adr_0002}
+
+Date: 2020-11-20
+
+## Status
+
+proposed
+
+## Context
+
+We need to determine a place to host our new release artifacts (tarballs). 
+Currently releases are being served by the Extension Distributor and the release branches of the git repository. 
+
+## Github
+
+#### pros: 
+
+- The same framework we use to produce the artifacts
+- Minimal effort required.
+
+#### cons:
+
+- Not a Foundation service
+- Needs to use the github frontend or routing with a token to get the artifacts (unless it's made into a release)
+
+## releases.wikimedia.org
+
+#### pros: 
+
+- Unified front for releases of the Wikimedia Foundation.
+
+#### cons:
+
+- Poor documentation (https://wikitech.wikimedia.org/wiki/Releases.wikimedia.org) for what we are looking for.
+- Seemingly bound to puppet/modules/releases repository for configuration
+- No direct control. 
+- Does not seem to be hosting any other extensions (ExtensionDistributor seems to be the desired place for these)
+
+## Wikiba.se
+
+#### pros: 
+
+- WMDE owned means direct control
+- The official site for Wikibase 
+
+#### cons:
+
+- We would need to build something that either pulls the artifacts or gets them uploaded from the pipeline. 
+- Effort required is unclear.
+
+## Extension Distributor
+
+#### pros: 
+
+- Seems to be the goto place for mediawiki extensions
+
+#### cons:
+
+- Looking through the code we would need to adapt it to serve our needs which might be undesired.
```

Comment:
I would add that I think this it totally antagonal to what extension distributor is supposed to be: AFAIK it is a release pipeline which produces the tarball to download, while we just want a "stupid" hosting service here, were we would push our (otherwise produced) tarballs into.

Reply:
I added another line but not sure if that matches what you was looking for, please have another look.

```
- Does not align with our needs of simple a hosting space for artifacts.

```